guide/src/StyleGuide.js

```diff
@@ -141,7 +141,7 @@ export default React.createClass({
             main: {
                 background: "whitesmoke",
                 width: "100%",
-                margin: "50px 0px 50px 250px", 
+                margin: "50px 0px 0px 250px", 
```

Comment:
Explain the auto margin, this seemed to be the source of the error I was addressing by having defined widths. I think this is related to the fact that the three major page parts have fixed positioning.